hooks/useServicesListsHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from "react"
+import { useQuery } from "react-query"
+import { Column } from "react-table"
+import { getServicesLists } from "../api"
+import { ServicesList } from "../models"
+
+export interface ServicesListsHandler {
+  servicesListsData: object[]
+  servicesListsColumns: Column[]
+  isLoading: boolean
+}
+
+export const useServicesListsHandler = (): ServicesListsHandler => {
```

Comment:
Creating [custom React hooks](https://reactjs.org/docs/hooks-custom.html) to contain most of the logic for a component/page I think is a pretty common pattern - at least it's what we do at work